Runtime/Game/Requests/ClassRequests.cs

```diff
@@ -325,7 +325,15 @@ public static void UnEquipIdAssetToClass(LootLockerGetRequest lootLockerGetReque
         {
             EndPointClass endPoint = LootLockerEndPoints.unEquipIDAssetToClass;
 
-            string getVariable = string.Format(endPoint.endPoint, lootLockerGetRequest.getRequests[0]);
+            string getVariable = "";
+            if(lootLockerGetRequest.getRequests.Count > 1)
+            {
+                getVariable = string.Format(endPoint.endPoint, lootLockerGetRequest.getRequests[0], lootLockerGetRequest.getRequests[1]);
+            }
+            else
+            {
+                getVariable = string.Format(endPoint.endPoint, lootLockerGetRequest.getRequests[0]);
```

Comment:
Shouldn't this be: string.Format(endPoint.endPoint, lootLockerGetRequest.getRequests[0], "");?
Wouldn't this make it fail when sending one parameter since the endpoint wants 2?
I could be wrong, if so merge, otherwise do the above fix.

Reply:
I have no idea actually. There were two usage places for this method, one which invoked with 1 argument and one with 2 so I did the quickest possible thing and made the 2 arguments one hopefully work. Didn't have time to test it out though. But you're probably right. Will check a little bit deeper tomorrow.

Reply:
Fixed now @JohannesLoot